### DISCUSSION TITLE
Reactive props generate an exception

### DISCUSSION THREAD

**Discussion by vincentdev15** [livewire/livewire#6156] [7. Bugs (archived)]
### Livewire version

3 beta 6

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a Livewire component with a form.
2. Create a Livewire component with a select / options field.
3. Use the select / options field in the form.
4. When I try to select an option in the dropdown, I get an error.

Cannot mutate reactive prop [product_id] in component: [components.products-select]*

```
<div class="mt-3 flex flex-col gap-4">
    <livewire:components.products-select wire:model.live="form.product_id"></livewire:components.products-select>
{{ 'Product id : '.$form->product_id }}
</div>

```

```
<?php

namespace App\Livewire\Components;

use App\Models\Product;
use Livewire\Attributes\Reactive;
use Livewire\Component;

class ProductsSelect extends Component
{
    #[Reactive] 
    public $product_id = null;

    public function render()
    {
        $products = Product::orderBy('name')->get();

        return view('livewire.components.products-select', compact('products'));
    }
}

```
### Additional information

I wondered if it was possible to use both reactive and modelable attributes, but it doesn't help to solve this problem.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@vincentdev15 reactive props can't be modified from the child component. I think what you want is probably a modelable prop - see https://livewire.laravel.com/docs/nesting#binding-to-child-data-using-wiremodel

Hope this helps!
